pkgs/shells/zsh/agdsn-zsh-config/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, zsh
+, coreutils
+, inetutils
+, procps
+}:
+
+with lib;
+
+stdenv.mkDerivation rec {
```

Comment:
`stdenvNoCC.mkDerivation` is more appropriate for packages not needing a compiler.